### ISSUE TITLE
Building hours update for Stav Hall

### ISSUE THREAD

**Issue by hawkrives** [StoDevX/AAO-React-Native#1310]
## Before:

```yaml
name: Stav Hall
image: stav
category: Food

schedule:
- title: Breakfast
  hours:
    - {days: [Mo, Tu, We, Th, Fr, Sa], from: '7:00am', to: '9:45am'}
    - {days: [Su], from: '8:30am', to: '10:15am'}

- title: Lunch
  hours:
    - {days: [Mo, Tu, We, Th, Fr], from: '10:30am', to: '2:00pm'}
    - {days: [Sa, Su], from: '11:00am', to: '1:30pm'}

- title: Dinner
  hours:
    - {days: [Mo, Tu, We, Th, Fr, Sa, Su], from: '4:30pm', to: '7:30pm'}

breakSchedule:
fall: []
thanksgiving: []
winter: []
interim: []
spring: []
easter: []
summer: []
```

## After:

```yaml
name: Stav Hall
image: stav
category: Food

schedule:
- title: Lunch
  hours:
    - {days: [Mo, Tu, We, Th, Fr], from: '10:30am', to: '2:00pm'}
    - {days: [Sa, Su], from: '11:00am', to: '1:30pm'}

- title: Dinner
  hours:
    - {days: [Mo, Tu, We, Th, Fr, Sa, Su], from: '4:30pm', to: '7:30pm'}

breakSchedule:
fall: []
thanksgiving: []
winter: []
interim: []
spring: []
easter: []
summer: []
```

**Comment by hawkrives** [comment]
The only thing that would be nicer would be to have it automatically diff them in this body... but I'd rather not bring in another runtime dep just for that, so how's this look?

Cc @rye @elijahverdoorn @drewvolz

**Comment by elijahverdoorn** [comment]
Yeah, I think that's good. Is there a way to auto-label them as "User Submission" or something so that we can filter?

**Comment by hawkrives** [comment]
Well, it should have automatically labeled this one as data/hours, so... it'd probably be easiest to just edit the title.

**Comment by hawkrives** [comment]
The current title is auto-generated - "Building hours update for $building"

**Comment by hawkrives** [comment]
Ah wait, it's still not indenting properly. See the things under the breakSchedules key? I'll have to look into it again.

**Comment by drewvolz** [comment]
@hawkrives I think you solved this with #1312 ?

**Comment by hawkrives** [comment]
I think so. Let me update to the new build and I'll check.

**Comment by hawkrives** [comment]
Yep. Closed by #1312.
